pkg/azure/azureconfig/config.go

```diff
@@ -1,6 +1,17 @@
 package azureconfig
 
 type Config struct {
-	// Env is the environment variable from which the connection string should be loaded.
-	Env string
+
+	// EnvConnectionString is the environment variable from which the connection
+	// string should be loaded.
+	EnvConnectionString string `json:"env"`
```

Comment:
Do we want to create a new `Env` struct to house various fields (`ConnectionString`, `ServicePrincipalPrefix`, etc.?)  Would then be accessed via `cfg.Env.ConnectionString`, etc.

Reply:
Let me think about this more and address improving the config in a follow-up PR. I'd like to streamline this as much as possible, right now the config struct is shared which is part of why things look awkward. 🤔